pkgs/applications/version-management/git-and-tools/gitstatus/default.nix

```diff
@@ -86,5 +86,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/romkatv/gitstatus";
     license = licenses.gpl3Only;
     maintainers = with maintainers; [ mmlb hexa SuperSandro2000 ];
+    platforms = platforms.all;
```

Comment:
Maybe git.meta.platforms would be more correct but makes no difference in reality.